packages/tree-widget/src/components/TreeWidgetControl.tsx

```diff
@@ -0,0 +1,96 @@
+import {
```

Comment:
Missing copyright header.  The CI should catch this, I'll see why it's not...

Reply:
fixed